slides/lesson2/solution/main.tf

```diff
@@ -1,14 +1,14 @@
 // Configure AWS Cloud provider
 provider "aws" {
-  region = "${var.aws_region}"
+  region = var.aws_region
 }
 
-terraform {
-  backend "s3" {
-    bucket = "wizeline-academy-terraform"
-    region = "us-east-2"
-  }
-}
+# terraform {
+#  backend "s3" {
+#    bucket = "your-bucket-name-here"
+#    region = "us-east-1"
```

Comment:
Same as above, let's change this to `us-east-2`

Reply:
let's keep as is, the participants should decide what region to use.